cws-installer/src/main/java/jpl/cws/task/CwsInstaller.java

```diff
@@ -169,7 +169,10 @@ public class CwsInstaller {
 	private static String cws_security_filter_class;
 	private static String startup_autoregister_process_defs;
 	private static String cws_token_expiration_hours;
+	private static String elasticsearch_protocol;
+	private static String elasticsearch_protocol_temp;
```

Comment:
change temp to initial

Reply:
Changed to:  `elasticsearch_protocol_init`, `elasticsearch_host_init`